src/Elements/Heading.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { css, withStyles } from "../withStyles";
+
+const Heading = ({ type, color = "default", styles, ...props }) => {
+  const HeadingTag = `${type}`;
```

Comment:
Might change this to `h${size}` instead of `${type} for more consistency between components and prevent confusion.